### DISCUSSION TITLE
Comparing 2 images and comparing prompt infos  etc....---is non existing ...but it is advertised that it can do this

### DISCUSSION THREAD

**Discussion by Akossimon** [zanllp/infinite-image-browsing#726] [Q&A] [answered]
Would anyone know how do  image comparison ?

its is advertised it can do these things:

A) Provides a side-by-side comparison of two images.
B)Provides a comparison of image generation information at the same time.

Yet i cannot find a way to do this, nor can i find anyone discussing it here neither.

can anyone help?

Thanks

**Comment by zanllp** [comment, marked as answer]
<img width="747" alt="image" src="https://github.com/user-attachments/assets/91dd5ce4-5950-4fe6-8f64-7463b6b5128a">


![image](https://github.com/user-attachments/assets/2f6c371a-354d-40b7-a5e4-c5227ce3776f)

**Comment by akos111** [comment]
does not work as you show it, dragging an image into these folders does NOT open them up at all. Seesm top be a function that was never developed, unless You have a newer version than i have

**Comment by zanllp** [comment]
@Akossimon @akos111 

This functionality has been supported for more than half a year.

https://github.com/user-attachments/assets/4872ea90-837e-4811-83ad-15b2de615633

**Comment by Akossimon** [comment]
i did it the wrong way, i dragged the image from the outside folder the image is located in, 

thanks so much for showing how it works
